js/src/components/contentAnalysis/ReadabilityAnalysis.js

```diff
@@ -35,26 +35,19 @@ class ReadabilityAnalysis extends React.Component {
 ReadabilityAnalysis.propTypes = {
 	results: PropTypes.array,
 	marksButtonStatus: PropTypes.string,
+	hideMarksButtons: PropTypes.bool,
 };
 
 /**
  * Maps redux state to ContentAnalysis props.
  *
  * @param {Object} state The redux state.
- * @param {Object} ownProps The components own props.
  *
  * @returns {Object} Props that should be passed to ContentAnalysis.
  */
-function mapStateToProps( state, ownProps ) {
```

Comment:
Why has this been removed?
The marker is not disabled when switching to text-mode anymore.